src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -465,8 +464,6 @@ public void editEntryAndFocusField(BibEntry entry, Field field) {
     }
 
     private void createMainTable() {
-        bibDatabaseContext.getDatabase().registerListener(SpecialFieldDatabaseChangeListener.INSTANCE);
```

Comment:
This listener is no longer used?

Reply:
SpecialFieldDatabaseChangeListener was part of the autosync complex. we removed it last year, but there were still some artifacts floating around.